RDKShell/RDKShell.cpp

```diff
@@ -3835,6 +3851,12 @@ namespace WPEFramework {
                     }
                 }
 
+                //Reset app suspended/hibernated for launch
```

Comment:
Might be good to use the #ifdef HIBERNATE_SUPPORT_ENABLED everywhere we use suspendOrHibernateApps array and mutex.